### ISSUE TITLE
Can any get a substitute for COMtypes for mac

### ISSUE THREAD

**Issue by Rohit-huta** [Rohit-huta/COMTYPES#1]
**### How can I replace COMtypes for mac, because according to my research COMtypes are for Windows.**

`from ctypes import cast, POINTER` 
`from comtypes import CLSCTX_ALL`
`from pycaw.pycaw import AudioUtilities, IAudioEndpointVolume`

`devices = AudioUtilities.GetSpeakers()`
`interface = devices.Activate(`
`    IAudioEndpointVolume._iid_, CLSCTX_ALL, None)`
`volume = cast(interface, POINTER(IAudioEndpointVolume))`

`print(volume.GetVolumeRange())`

ERROR appears:
`ImportError: cannot import name 'COMError' from '_ctypes'`

**Comment by AndreMiras** [comment]
Yeah pycaw is for Windows indeed. I don't know much about macOS. And I actually don't know much about Windows either.
I built pycaw for a friend, but I'm a Linux user

**Comment by Rohit-huta** [comment]
> Yeah pycaw is for Windows indeed. I don't know much about macOS. And I actually don't know much about Windows either.
> I built pycaw for a friend, but I'm a Linux user

Yeah! that's great, just so if you can any solution for it plz ping me up.
Thank you
